File: ChakaiP2EAPI/ChakaiP2EFunctionsAPI/Models/ChakaiSchema.cs
using ChakaiP2EFunctionsAPI.Utilities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ChakaiP2EFunctionsAPI.Models
{
    [View, Plural("ChakaiSchema")]
    internal class ChakaiSchema : BaseModel
    {
        [NoInsert, NoUpdate]
        public int ChakaiSchemaId { get; set; }
        [NoInsert, NoUpdate]
        public string ChakaiSchemaName { get; set; }
        [NoInsert, NoUpdate]
        public string ChakaiSchemaShortName {  get; set; }
        [NoInsert, NoUpdate]
        public string ChakaiSchemaTable { get; set; }

    }
}

File: ChakaiP2EVue/chakaip2evueapp/src/types/chakaischema.ts
export interface ChakaiSchema {
    ChakaiSchemaId: number;
    ChakaiSchemaName: string;
    ChakaiSchemaShortName: string;
    ChakaiSchemaTable: string;
}

export const COST_ALIASES: Record<string, string> = {
    '1 Action': '🔷',
    '2 Action': '🔷🔷',
    '3 Action': '🔷🔷🔷',
    'Reaction': '↺',
    'Free Action': '◇',
};
